tests/unit/tests.py

```diff
@@ -0,0 +1,15 @@
+import unittest
```

Comment:
If you look at `master`, i was having a hard time getting these tests to run, so I moved them to be under `phc/tests/subject_search_test.py` to make the importing work.

Reply:
Whoops, I must have resolved the conflict incorrectly. Will take a look.